src/index.ts

```diff
@@ -209,4 +210,49 @@ app.post("/attendance", async (c) => {
   return new Response();
 });
 
+app.get("/attendances/:course_id", async(c) => {
+  const subjectID = c.req.param('course_id');
+  const honoSession = c.get('session');
+  const session = honoSession.get("login") as Session | null;
+  if ( !session ){
+    return c.text("Unauthorized", 401);
+  }
+
+  //科目が存在するか、履修しているかをチェック
+  const [subjectResult, registration] = c.env.DB.batch([
```

Comment:
```suggestion
  const [subjectResult, registration] = await c.env.DB.batch([
```
`Promise` が `await` されていないので分割代入で取り出せません.